src/CoreShop/Bundle/IndexBundle/Installer.php

```diff
@@ -15,16 +15,17 @@
 namespace CoreShop\Bundle\IndexBundle;
 
 use Pimcore\Console\Application;
-use Pimcore\Extension\Bundle\Installer\InstallerInterface;
+use Pimcore\Extension\Bundle\Installer\SettingsStoreAwareInstaller;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Output\NullOutput;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\HttpKernel\KernelInterface;
 
-class Installer implements InstallerInterface
+class Installer extends SettingsStoreAwareInstaller
```

Comment:
what about the migration? bundle should be marked installed with a migration

Reply:
Why? Just do a `parent::install()` in your `install()` method which will execute the initial setting-store stuff.

Reply:
I am talking about already installed IndexBundles. With this change, these would not be marked as installed anymore

Reply:
True... I think we can then ignore it

Reply:
Sorry, just to make it clear: By "ignore it" you mean, remove the migration? 😅

Reply:
@aarongerig Since you added it, we can keep it now :) thanks for that